src/components/auction/OrderPlacement/index.tsx

```diff
@@ -277,31 +277,26 @@ const OrderPlacement: React.FC<OrderPlacementProps> = (props) => {
     [derivedAuctionInfo?.auctionEndDate, derivedAuctionInfo?.orderCancellationEndDate],
   )
 
-  const isLoading = React.useMemo(
-    () => !auctionInfo || (auctionInfo && auctionInfo.auctionInfoLoading),
-    [auctionInfo],
-  )
-  const isPrivate = React.useMemo(
-    () => auctionInfo && auctionInfo.auctionDetails && auctionInfo.auctionDetails.isPrivateAuction,
-    [auctionInfo],
-  )
+  const isPrivate = React.useMemo(() => auctionDetails && auctionDetails.isPrivateAuction, [
+    auctionDetails,
+  ])
   const signatureAvailable = React.useMemo(() => signature && signature.length > 10, [signature])
   const isPlaceOrderDisabled =
     !isValid || notApproved || showWarning || showWarningWrongChainId || showConfirm
 
   return (
     <>
       <Wrapper>
-        {isLoading && <InlineLoading size={SpinnerSize.small} />}
-        {!isLoading && isPrivate && !signatureAvailable && (
+        {auctionInfoLoading && <InlineLoading size={SpinnerSize.small} />}
```

Comment:
Also here, this will reintroduce quite some flashings

Reply:
Done!